File: fizzbuzz.py
# Description: FizzBuzz program


def fizzBuzz(n: int):
    output = []
    string = ""
    if n % 3 == 0:
        string = string + "Fizz"
    if n % 5 == 0:
        string = string + "Buzz"
    if string == "":
        string = n
    output.append(string)
    string = ""
    return output


for i in range(1, 101):
    print(fizzBuzz(i))

File: fizzbuzz.dart

String FizzBuzz(int n){
  var Output = "";

  if (n % 3 == 0) {
    Output = Output + "Fizz";
  }
  if (n % 5 == 0) {
    Output = Output + "Buzz";
  }
  if (Output == ""){
    Output = n.toString();
  }

  return Output;
}

void main(){
  print(FizzBuzz(5));
  print(FizzBuzz(15));
  print(FizzBuzz(25));
  print(FizzBuzz(7));
}